build/generic/web/viewer.js

```diff
@@ -9654,6 +9654,21 @@ function setupPageNumberDisplay(eventBus) {
   eventBus.on('pagechange', function(ev) {
     pageNumberDisplay.innerText = ev.pageNumber;
   });
+
+  // Fade the page number display out when the user is not scrolling. This
+  // only applies to the mobile layout.
+  var timeout = null;
+  var pageNumberInfo = document.getElementById('pageNumberInfo');
+  UIUtils.watchScroll(document.getElementById('viewerContainer'), function() {
+    if (timeout) {
+      pageNumberInfo.className = '';
+      clearTimeout(timeout);
+    }
+
+    timeout = setTimeout(function() {
+      pageNumberInfo.className = 'fadeOut';
```

Comment:
I was trying to support older browsers. Not sure if it's necessary any more though. What do you think?